tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -94,6 +94,12 @@ Verify Documentation Link HTTP Status Code
     ${links}=  Get RHODS Documentation Links From Dashboard
     Check External Links Status     links=${links}
 
+Verify GPU Items Appears In Resources Page
+    [Tags]  ODS-1226
```

Comment:
How about **Search And Verify GPU Items Appears In Resources Page** as keyword name .

Reply:
Yes that sounds more specific.